games-list/src/components/GamesCompleted.tsx

```diff
@@ -20,13 +20,34 @@ export default function GamesCompleted({
   gamesList,
   setList,
 }: GameFieldProps) {
+  const [emptyChecker, setChecker] = useState(true); // checks if mongoDB collection returned an empty list
+
+  // receives data from mongoDB and sets gamesList if it exists in DB
+  async function getData() {
+    const response = await fetch("/api/getData", {});
+    if (response.ok) {
+      const mongoData = await response.json();
+      if (mongoData[0] !== undefined) {
+        setList(mongoData[0].mongoList); //holds previously saved data
+        setChecker(false); // gamesList exists in DB
+      }
+    } else {
+      alert("ERROR!!");
+    }
+  }
+  //used to load data from mongoDB once on page load
+  useEffect(() => {
+    getData();
+  }, []);
```

Comment:
Code is self explanatory I think. :)

Reply:
Removed it :D . Finding out how to do it the first time seemed very unique, so wanted to keep it for future reference. But now I'm more comfortable with the useEffect hook.